### PR TITLE
Versioned docs

### PR DESCRIPTION
**Pull request by lucperkins**
Fixes #151

### CODE REVIEW

**Comment by tomkerkhove** [comment]
Thanks man!

If I get it correctly we can just add folders with new versions and the dropdown will list the versions, right? Nice!

Can we add a new version for just one file without the others?

File: Makefile [hunk 1]
```diff
@@ -12,17 +12,17 @@ serve: yarn
 production-build: clean
 	hugo \
 		--minify
-	
-	make check-links
+
+	#make check-links
```
[the remark this answers is not in the document]
Comment (lucperkins) [reply Makefile:16]: Yes. See #154.

**Comment by lucperkins** [comment]
@tomkerkhove Yes, you just need to add a new folder and update the `config.toml`. And no, versions don't apply to specific docs.

**Comment by tomkerkhove** [comment]
> @tomkerkhove Yes, you just need to add a new folder and update the `config.toml`.

Maybe good to add a small section in README then?
 
> As for the versioning system, versions apply to all docs, not to specific docs, as requested.

So we have to copy everything then? That's fair.

**Comment by tomkerkhove** [comment]
We'll discuss this in our standup on Thrusday as a final sanity check, ok for you?

For me it definately looks good!

**Comment by tomkerkhove** [comment]
In general we like hte approach but are wondering in terms of minor changes, what would that look like?

Would you mind adding some pointers in the docs around that? Do we have to copy the whole folder even only 1 scaler changed or how does that work?

**Comment by lucperkins** [comment]
That’s up to you. You can make minor changes to the docs within a specific version if you’d like, but if there’s a new version you do have to copy the directory. There really isn’t any other way to do that.

**Comment by tomkerkhove** [comment]
Thoughts @jeffhollan @zroubalik @ahmelsayed ?

**Comment by tomkerkhove** [comment]
https://github.com/kedacore/keda-docs/pull/151/files#diff-04c6e90faac2675aa89e2176d2eec7d8R84

This is how new docs are added for a version @jeffhollan

**Comment by tomkerkhove** [comment]
@lucperkins Is this PR ready to go?

**Pull request merged**